pyproject.toml

```diff
@@ -297,6 +297,7 @@ module = [
     "sentry.sentry_apps.installations",
     "sentry.sentry_metrics.indexer.postgres.postgres_v2",
     "sentry.shared_integrations.client.proxy",
+    "sentry.snuba.entity_subscription",
```

Comment:
this should never be added to only removed -- please find what's regressed here and fix it (likely the caller is in an ignored file and the signature is incorrect)